cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
The `debug_toolbar` should only be included in development environments. Make sure to conditionally include this path based on the `DEBUG` setting to avoid exposing debug information in production.